.release.cloudbuild.yaml

```diff
@@ -116,229 +116,6 @@ steps:
   id:   'copyPythonSDKToLatest'
   waitFor: ['copyPythonSDKLocal']
 
-# Pull and retag the images for the Dataflow-based pipeline components
```

Comment:
Why not do all the job in the release yaml in the release script?

Reply:
the system images doesn't have the double release problem.